docs/standard/serialization/system-text-json/migrate-from-newtonsoft.md

```diff
@@ -518,7 +566,24 @@ During deserialization, `Newtonsoft.Json` adds objects to a collection even if t
 
 ### Snake case naming policy
 
-The only built-in property naming policy in System.Text.Json is for [camel case](customize-properties.md#use-camel-case-for-all-json-property-names). `Newtonsoft.Json` can convert property names to snake case. A [custom naming policy](customize-properties.md#use-a-custom-json-property-naming-policy) can provide this functionality. For more information, see GitHub issue [dotnet/runtime #782](https://github.com/dotnet/runtime/issues/782).
+:::zone pivot="dotnet-8-0"
+
+System.Text.Json includes a built-in naming policy for snake case. However, there are some behavior differences with `Newtonsoft.Json` for some inputs. The following table shows some of these differences when using the <xref:System.Text.Json.JsonNamingPolicy.SnakeCaseLower?displayProperty=nameWithType> policy.
+
+| Input          | Newtonsoft.Json  | System.Text.Json |
+|----------------|------------------|------------------|
+| "AB1"          | "a_b1"           | "ab1"            |
+| "abc123DEF456" | "abc123_de_f456" | "abc123_def456"  |
+| "ABC123def456" | "ab_c123def456"  | "abc123def456"   |
```

Comment:
Perhaps a more real-world example here could be `SHA512Managed`, which renders as `sha512_managed` in STJ and as `sh_a512_managed`